.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,69 @@
+name: PHP Tests
+
+on: push
+
+jobs:
+  test:
+    name: WP ${{ matrix.wp }} and PHP ${{ matrix.php }}
+    # Ubuntu-20.x includes MySQL 8.0, which causes `caching_sha2_password` issues with PHP < 7.4
+    # https://www.php.net/manual/en/mysqli.requirements.php
+    # TODO: change to ubuntu-latest when we no longer support PHP < 7.4
+    runs-on: ubuntu-16.04
+    continue-on-error: ${{ matrix.allowed_failure }}
+
+    env:
+      WP_VERSION: ${{ matrix.wp }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4' ]
+        wp: [ '5.6', '5.7' ]
+        allowed_failure: [ false ]
+        include:
+          - php: '8.0'
+            wp: '5.6'
+            allowed_failure: true
+          - php: '8.0'
+            wp: '5.7'
+            allowed_failure: true
+          # PHP nightly.
+          - php: '8.1'
+            wp: '5.7'
+            allowed_failure: true
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - name: Setup PHP ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: pcov
+          # https://make.wordpress.org/hosting/handbook/handbook/server-environment/#php-extensions
+          extensions: curl, dom, exif, fileinfo, hash, json, mbstring, mysqli, libsodium, openssl, pcre, imagick, xml, zip
+
+      - name: Install Composer dependencies (PHP < 8.0 )
+        if: ${{ matrix.php < 8.0 }}
+        uses: ramsey/composer-install@v1
+
+      - name: Install Composer dependencies (PHP >= 8.0)
+        if: ${{ matrix.php >= 8.0 }}
+        uses: ramsey/composer-install@v1
+        with:
+          composer-options: --ignore-platform-reqs
+
+      - name: Start MySQL service
+        run: sudo systemctl start mysql.service
+
+      - name: Install WordPress test site
+        run: bash bin/install-wp-tests.sh wordpress_test root root localhost
```

Comment:
In line 21, I see that different WP versions being specified. However, `bin/install-wp-tests.sh wordpress_test root root localhost` would always install the latest WP version. I'd change that line to `bin/install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wp }}` to install different WP versions for the tests.

Reply:
Thanks @nielslange. You're correct. I addressed this issue on 8688a81